setup.cfg

```
@@ -17,5 +17,5 @@
 [options.entry_points]
 cylc.configure = 
     configure = cylc.rose.rose:get_rose_vars
```

This name needs to be unique so best to name it `rose` or `cylc_rose` to avoid conflicts with any other plugins:

```suggestion
    rose = cylc.rose.rose:get_rose_vars
```